code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -89,36 +89,41 @@
 		beaker = null
 
 /obj/machinery/atmospherics/components/unary/cryo_cell/update_icon()
+
 	cut_overlays()
 
 	if(panel_open)
 		add_overlay("pod-panel")
 
-	if(state_open)
+	if(state_open && (icon_state != "pod-open"))
+		animate(src, icon_state = "pod-anim1", icon_state = "pod-anim2", time = 2)
 		icon_state = "pod-open"
 		return
 
+	if(!state_open && (icon_state == "pod-open"))
+		animate(src, icon_state = "pod-anim2", icon_state = "pod-anim1", time = 2)
+
 	if(occupant)
 		var/image/occupant_overlay
 
 		if(ismonkey(occupant)) // Monkey
-			occupant_overlay = image(CRYOMOBS, "monkey")
+			occupant_overlay = mutable_appearance(CRYOMOBS, "monkey")

```

Comment:
These need to be `image` instead of `mutable_appearances` so that `dir` can be independent of parent. Please fix this and add a comment so that no one will do this again.

Reply:
Currently it follows dir of the cryo even if its image

Reply:
Wrong. Line 123 (or 128): `occupant_overlay.dir = SOUTH`

Reply:
Disregard this i was on the old version of byond so it did follow the locations dir even if it was set manually

Reply:
You removed all `occupant_overlay =...` lines now, you need to replace em

Reply:
Im not sure how did that happen, they were present in the dme

Reply:
Please add a comment in the code so that no one will try to replace `image` again later on